examples/stable_diffusion_xl/gm/modules/diffusionmodules/guiders.py

```diff
@@ -59,30 +58,28 @@ def prepare_inputs(self, x, s, c, uc):
 class LinearPredictionGuider(nn.Cell):
     def __init__(
         self,
-        max_scale: float,
-        num_frames: int,
         min_scale: float = 1.0,
+        max_scale: float = 2.5,
         additional_cond_keys: Optional[Union[List[str], str]] = None,
     ):
         super().__init__()
         self.min_scale = min_scale
         self.max_scale = max_scale
-        self.num_frames = num_frames
-        self.scale = Tensor(np.expand_dims(np.linspace(min_scale, max_scale, num_frames, dtype=np.float32), 0))
 
         additional_cond_keys = additional_cond_keys or []
         if isinstance(additional_cond_keys, str):
             additional_cond_keys = [additional_cond_keys]
         self.additional_cond_keys = additional_cond_keys
 
-    def construct(self, x: Tensor, sigma: Tensor) -> Tensor:
+    def construct(self, x: Tensor, sigma: Tensor, num_frames: int) -> Tensor:
```

Comment:
`num_frames` does not belong to sdxl, maybe move out to mindone, or a separate examples

Reply:
Yes, `LinearPredictionGuider` is specific to SVD. Moved it there.